### ISSUE TITLE
Possible windows CI outage, starting 2016-04-12 around 17:35 EDT

### ISSUE THREAD

**Issue by david-german-tri** [RobotLocomotion/drake#2075]
After #2069, the continuous win32 and win64 builds went red.  Example from win32:

https://drake-jenkins.csail.mit.edu/job/windows-msvc-32-continuous-release/114/console

I'm going to test a rollback of #2069.  It's hard to believe that a Drake code change caused these particular errors, though, so @jamiesnape FYI.

**Comment by david-german-tri** [comment]
#2069 was exonerated (see #2076).  It appears whatever happened here was transient.

**Comment by jamiesnape** [comment]
Duplicate of #2035.
